ASTProcessor/src/main/java/edu/usb/argos/ASTProcessor/visitor/core/entities/classes/AnnotationInfo.java

```diff
@@ -0,0 +1,30 @@
+package edu.usb.argos.ASTProcessor.visitor.core.entities.classes;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class AnnotationInfo {
```

Comment:
use Lombok annotations for getters and setters, and the same thing with other entities

Reply:
Done